components/Registration/index.tsx

```diff
@@ -65,6 +65,7 @@ const Registration: FunctionComponent<RegistrationProps> = ({
 }) => {
```

Comment:
Please add a meaningful description to the PR. Also reference the issue number

Reply:
Ok sir